### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by thokkat**

### CODE REVIEW

File: game/world/aiqueue.cpp [hunk 1]
```diff
@@ -184,7 +184,7 @@ AiQueue::AiAction AiQueue::aiEquipBestArmor() {
   return a;
   }
 
-AiQueue::AiAction AiQueue::aiEquipBestMeleWeapon() {
+AiQueue::AiAction AiQueue::aiequipBestMeleeWeapon() {
```
Comment (Try) [inline game/world/aiqueue.cpp:187]: ai**E**quipBestMeleeWeapon

**Comment by Try** [comment]
HI, @thokkat , thanks for PR!

Can you please fix that one typo in PR)

**Comment by thokkat** [comment]
Done. Good catch :)

**Comment by Try** [comment]
Merged, thanks!

**Pull request merged**